### ISSUE TITLE
Editor disconnect/reconnect

### ISSUE THREAD

**Issue by user9319062** [TomWhitwell/8mu_Public#4]
With the web editor open I am only able to edit for a few seconds before it drops, reconnects, and I lose settings before being able to save them.

Steps to recreate
- Editor > Edit Config
- Change a setting
- Don't click to update, just wait

Observed
The unit disconnects, maddeningly, within about 10 seconds. Saving works if you're quick enough, however you can't change more than maybe 2 settings at once before it dumps them

video: https://drive.google.com/file/d/1fW8w71i0r9V7ep8wCjpk2rD4xhKosvxm/view?usp=share_link (apologies if google still show 360p vid)

In Inspect I can see it's listening in a loop like this:
```
Attached sysex listener to  Music Thing m0 Plus
logger.js:3 Hearbeat requesting config.
logger.js:3 Requesting config over Music Thing m0 Plus
logger.js:3 Hoping to receive on Music Thing m0 Plus
```

Then when it drops, we see this:
```
it's a c0nFig message
logger.js:3 Generating config from Uint8Array(122) [240, 125, 0, 0, 15, 4, 1, 0, 1, 1, 0, 0, 0, 15, 0, 122, 31, 1, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 6, 35, 10, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 8, 9, 10, 11, 12, 13, 14, 41, 42, 43, 44, 45, 46, 47, 48, 49, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, …]
logger.js:3 Received config ae {faderBlink: 1, accelBlink: 0, controllerFlip: 0, deviceId: 4, firmwareVersion: '1.0.1', …}
```

This is the same log message (I think) as when you plug it in. However in this case it repeats 15/20 times (this appears instantaneous) before going back to cycling through the `Hearbeat` process again. Additionally, every LED flashes on quickly once (not animated, as when you power it up).  If the editor is closed, there is no activity on the LEDs (until you use it, obviously).

I tried numerous different USB-C cables but it's the same for each. Current workaround is to download the json file and manually edit each before uploading in order to preserve sanity

Tested on Mac OS Monterey, Chrome (latest) & Firefox (latest)

**Comment by TomWhitwell** [comment]
Hi, that sounds super frustrating. It's not something I've seen in beta testing. Aside from editing, is the 8mu reliable when sending out MIDI? 

Coincidentally I'm about to do a significant update the web editor - moving to v2 to match more recent 16n editor. This should happen in the next few days so will be useful to see if it has any impact - if you email me I can share a beta link.

**Comment by user9319062** [comment]
Yes it's reliable I think. I only built it yesterday, but so far so good, getting consistent results from the faders when connected - it's only updating that presents a problem

I still half-suspect it's my USB cables. Even though I've tried 3 different ones, none of them are 'brand new' (but they don't have any problems doing what they're normally used for, ie charging a phone, data transfer to ipad etc. 

I'll also try from a different computer when I get the chance. Will email you about that beta!
